pkgs/applications/virtualization/slirp4netns/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  name = "slirp4netns-${version}";
+  version = "0.3.0-alpha.2";
```

Comment:
What do you think of having two versions for this; one for v2 and v3?

Reply:
It would probably be easy to do, but I'd prefer to just merge this and upgrade to proper v3 whenever that's released.